index.css

```diff
@@ -0,0 +1,161 @@
+html,
+body {
+    margin: 0;
+    font-family: 'Karla', sans-serif;
+    background-color: #219ebc;
+}
+
+/*typography*/
+
+h1,
+h2,
+h3 {
+    text-align: center;
+}
+
+h1,
+h2 {
+    color: #EED6D3;
+    margin: 0;
+}
+
+h1 {
+    font-size: 28px;
+}
+
+h2 {
+    font-size: 16px;
+}
+
+h3 {
+    margin: 25px;
+}
+
+/* layout */
+
+header {
+    background-color: #67595E;
```

Comment:
same with this, I would make a class rather then putting a background-color on all headers by default

Reply:
Usually you only have one header though right? Should I apply a class to every element?